internal/install/execution/recipe_log_forwarder.go

```diff
@@ -51,15 +56,8 @@ func (lf *RecipeLogForwarder) PromptUserToSendLogs(reader io.Reader) bool {
 	return false
 }
 
-func (lf *RecipeLogForwarder) SendLogsToNewRelic(outputFilePath string, recipeName string) {
-	// open file, build log entries
-	f, err := os.Open(outputFilePath)
-	if err != nil {
-		log.Fatalf("open file error: %v", err)
-		return
-	}
-	defer f.Close()
-	lf.buildLogEntryBatch(recipeName, bufio.NewScanner(f))
+func (lf *RecipeLogForwarder) SendLogsToNewRelic(recipeName string, recipeOutput []string) {
```

Comment:
@mbruzina I like this method name, very clear. I would just rename the recipeOutput parameter as it seems to tie a specific use case to what that function can do, when it doesn't matter what kind of string array is passed in (it doesn't have to be a recipe output string array).